src/app-nav-bar/app-nav-bar.js

```diff
@@ -29,7 +31,17 @@ export default function AppNavBar(props: AppNavBarPropsT) {
     appDisplayName,
     appDisplayNameLink,
   } = props;
-
+  const activeMainNavItem = mainNav.find(item => item.active);
+  const activeSubNav =
```

Comment:
Maybe refactor this to a `let` + `if statement`? Ultimately resolving to a value due to subtle JS interactions leads to less quickly understandable code